canonical-websites/cn.ubuntu.com
# Upgrade to Vanilla framework v1.6.3

## Done
Updated Vanilla to v1.6.3. Remove un-needed styles and changed the old takeover to the default one.

## QA
- Pull code
- Run `./run`
- Go to http://localhost:8010/
- Check the site over

## Issue / Card
Fixes https://github.com/vanilla-framework/vanilla-framework/issues/1467
